### ISSUE TITLE
ReferenceError: regeneratorRuntime is not defined

### ISSUE THREAD

**Issue by jordiyapz** [kqito/use-tus#39]
The problem arise in `useAutoAbort` hook. After being bundled by babel, I suppose, the library throw this error.

```
Uncaught ReferenceError: regeneratorRuntime is not defined
    abortUploading index.esm.js:199
    useAutoAbort2 index.esm.js:226
    ....
```

**Comment by maclementED** [comment]
Did you find a solution for this ? Im having the same issue on NextJS

**Comment by kqito** [comment]
I'm sorry for the delay.
This bug fixed by #40.
Thank you for your report! ;)
